SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -725,6 +725,13 @@ int64 USpatialActorChannel::ReplicateActor()
 			const VirtualWorkerId NewAuthVirtualWorkerId = NetDriver->LoadBalanceStrategy->WhoShouldHaveAuthority(*Actor);
 			if (NewAuthVirtualWorkerId != SpatialConstants::INVALID_VIRTUAL_WORKER_ID)
 			{
+				// Update Actor SpawnData location for when new worker receives authority to prevent hysteresis.
+				SpatialGDK::SpawnData* ActorSpawnData = NetDriver->StaticComponentView->GetComponentData<SpatialGDK::SpawnData>(EntityId);
+				check(ActorSpawnData != nullptr);
+				ActorSpawnData->Location = Actor->GetActorLocation();
```

Comment:
Location should be set via 
```
		const USceneComponent* RootComponent = Actor->GetRootComponent();

		Location = RootComponent ? FRepMovement::RebaseOntoZeroOrigin(Actor->GetActorLocation(), Actor) : FVector::ZeroVector;
```
Could add a utility function to SpawnData to do this.